pkgs/racket-doc/scribblings/foreign/pointers.scrbl

```diff
@@ -212,11 +212,11 @@ see @|InsideRacket|.
                                       ctype?) 
                                 @#,elem{absent}]
                  [cptr cpointer? @#,elem{absent}]
-                 [mode (one-of/c 'raw 'atomic 'nonatomic 'tagged
-                                 'atomic-interior 'interior
-                                 'stubborn 'uncollectable 'eternal)
+                 [mode (or/c 'raw 'atomic 'nonatomic 'tagged
+                             'atomic-interior 'interior
+                             'stubborn 'uncollectable 'eternal)
                        @#,elem{absent}]
-                 [fail-mode (one-of/c 'failok) @#,elem{absent}])
+                 [fail-mode 'failok @#,elem{absent}])
```

Comment:
Not related to your change, but I wonder why these arguments don't have the contract `(or/c #f ...)` with the default value of `#f`.